### DISCUSSION TITLE
Feature Request: support for iceberg table format

### DISCUSSION THREAD

**Discussion by terekete** [duckdb/duckdb#7645] [Ideas]
Why do you want this feature?

Is there a way for duck db to utilize iceberg table format as an option backed on object storage. This as a native capability to DuckDB would be very useful to folks looking to do analytics on object storage while using the virtues of table formats such as upserts, evolutions, time tavel and likely cheaper data resources.

Is this possible ?

**Comment by Mause** [comment]
See https://github.com/duckdb/duckdb/discussions/1668
